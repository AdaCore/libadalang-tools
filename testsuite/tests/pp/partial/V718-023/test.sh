partial_gnatpp -P default.gpr -S main.adb -SL 5 -SC 7 -EL 5 -EC 17
partial_gnatpp -P default.gpr -S main.adb -SL 5 -SC 7 -EL 7
partial_gnatpp -P default.gpr -S main.adb -SL 5 -SC 5 -EL 5 -EC 5
partial_gnatpp -P default.gpr -S main.adb -SL 1 -EL 10

