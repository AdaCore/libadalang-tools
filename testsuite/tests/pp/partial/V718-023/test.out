*************************************
main.adb(<ObjectDecl ["A"] main.adb:5:4-5:16>) - 5:1-5:16
^
      A : Integer;
$
*************************************

*************************************
main.adb(<DeclBlock main.adb:4:4-8:11>) - 4:1-8:11
^
   declare
      A : Integer;
   begin
      null;
   end;
$
*************************************

*************************************
main.adb(<ObjectDecl ["A"] main.adb:5:4-5:16>) - 5:1-5:16
^
      A : Integer;
$
*************************************

*************************************
main.adb(<SubpBody ["main"] main.adb:1:1-10:10>) - 1:1-10:10
^
procedure main is
begin

   declare
      A : Integer;
   begin
      null;
   end;

end main;
$
*************************************

